tests/core/test_library.py

```diff
@@ -297,6 +298,109 @@ def test_search_normalises_bad_queries(self):
         )
 
 
+class GetDistinctTest(BaseCoreLibraryTest):
+    def test_with_query(self):
+        self.library1.get_distinct.return_value.get.return_value = {}
+        self.library2.get_distinct.return_value.get.return_value = {}
+
+        result = self.core.library.get_distinct("album", {"any": ["a"]})
+
+        self.library1.get_distinct.assert_called_with("album", {"any": ["a"]})
+        self.library2.get_distinct.assert_called_with("album", {"any": ["a"]})
+        assert set() == result
```

Comment:
This should be written as `assert result == set()` (rg. `assert actual == expected`)

Reply:
I've been trying to get myself into the habbit of putting something unassignable on the LHS and all of these are artefacts of that. Flake8 doesn't complain about this, which it presumably could spot. Is this style actally an issue? I not fussed either way.

Reply:
I'm also heavily leaning in the direction of `assert actual == expected`, just like @djmattyg007. I think we might have some asserts that are the other way around for historical reasons, but I usually fix them when I spot them.

Reply:
The convention when writing assertions in most languages is to put the actual value first, and the expected value second. I'm pretty sure pytest expects this ordering when displaying output too.

Also Python doesn't allow accidental assignments in these kinds of expressions (you'd need to use the walrus operator, which would automatically fail the tests in python3.8 and below). So I don't think there's really a need to use Yoda expressions in Python code.

Reply:
Don't get me started on the walrus.

Reply:
And I will flip these around when I get a few moments later today.